src/container/ListContainer.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from "react";
+import { ListContainerStyle } from "../assets/styles/List.style";
+import List from "../components/List";
+
+class ListContainer extends Component {
+  render() {
+    return (
+      <ListContainerStyle>
+        {this.props.list && this.props.list.length
```

Comment:
Use destructuring here instead of repeating this.props

Reply:
Done